extensions/amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html

```diff
@@ -37,6 +37,16 @@
      layout="responsive" width="480" height="270">
   </amp-kaltura-player>
 
+  <!-- valid kaltura ce example -->
+  <amp-kaltura-player
+    data-service-url="front.video.funke.press"
```

Comment:
nit, optional: alignment appears off on this line.

Reply:
fixed